native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/AbstractStatementParameterProcessor.java

```diff
@@ -197,81 +197,92 @@ protected abstract int setCustomBOpenRecord(Connection connection, PreparedState
                                                 Object value, boolean returnType) throws DataError, SQLException;
 
     public void setParams(Connection connection, PreparedStatement preparedStatement,
-                          Object[] insertions) throws DataError, SQLException {
+                          Object[] insertions, String query) throws DataError, SQLException {
```

Comment:
This will lead to an API change. IMO we don't need the query in the log since the user already has that info.